research/src/pages/light-dismiss.behaviors.mdx

```diff
@@ -0,0 +1,52 @@
+---
+menu: Behaviors
+name: Light Dismiss
+path: /behaviors/light-dismiss
+---
+
+import Image from '../components/image'
+
+# Light dismiss
+
+## Overview
+
+"Light dismiss" describes the behavior of a component that dismisses
+itself automatically when a user performs an action that implies that they are
+finished interacting with the component. Such actions include clicking outside
+of its bounds or pressing the escape key. "Dismissal" generally means that the
+component becomes hidden.
+
+Examples of components that often use light dismiss include a listbox input's
+dropdown, a date input with a calendar flyout, and many non-modal popup dialogs.
+
+<Image
+  src="select-light-dismiss.gif"
+  style="height: 30%; width: auto;"
+  alt="A cursor clicking a select control to open its listbox, and then clicking outside the control to close it."
+/>
+<Image
+  src="date-light-dismiss.gif"
+  style="height: 30%; width: auto;"
+  alt="A cursor clicking a date control to open a calendar flyout, and then clicking outside the control to close it."
+/>
+
+## Light dismiss triggers
+
+The actions that trigger light dismiss are:
+
+- The user presses the escape key.
+- A focus change occurs (because of either user interaction or script), where
+  the focus target is outside of the contents of the component. This includes
+  the case where the user invokes a non-focusable element, which causes focus to
+  switch to the `<body>`.
+  - There is one exception to this: if a user invokes a non-focusable element
```

Comment:
It moves either to the closest focusable ancestor of the clicked element, or to the `<body>` if there is no focusable ancestor. You can try this out with the rightmost `<select>` here: https://codepen.io/daniec/pen/poyPOKY.  Note that the `focusout` event causes the popup to collapse when the nonfocusable text inside it is clicked, because that click causes the focus to jump to the `<body>`.

This is a bit odd but it's just how focus works in the web platform.

That said, I talked about this with some Microsoft folks and we thought that it's probably better to pull the exception for this behavior outside of the core of the light dismiss definition, and instead include a note recommending that developers make the top level of the component with light dismiss focusable (e.g. with `tabindex="-1"`).  If this is done, then focus won't move outside of the light dismiss component and it won't be hidden.

I've made that change to this PR.